spotify_player/src/event/popup.rs

```diff
@@ -259,6 +267,100 @@ fn handle_command_for_queue_popup(
+                                name.push(c);
+                            }
+                            PlaylistCreateCurrentField::Desc => {
+                                desc.push(c);
+                            }
+                        }
+                        return Ok(true);
+                    }
+                    crossterm::event::KeyCode::Backspace => {
+                        match &current_field {
+                            PlaylistCreateCurrentField::Name => {
+                                if !name.is_empty() {
+                                    name.pop().unwrap();
+                                }
+                            }
+                            PlaylistCreateCurrentField::Desc => {
+                                if !desc.is_empty() {
+                                    desc.pop().unwrap();
+                                }
+                            }
+                        }
+                        return Ok(true);
+                    }
+                    crossterm::event::KeyCode::Tab => {
+                        *current_field = match &current_field {
+                            PlaylistCreateCurrentField::Name => PlaylistCreateCurrentField::Desc,
+                            PlaylistCreateCurrentField::Desc => PlaylistCreateCurrentField::Name,
+                        };
+                        return Ok(true);
+                    }
+                    crossterm::event::KeyCode::Enter => {
+                        client_pub.send(ClientRequest::CreatePlaylist {
+                            playlist_name: name.to_owned(),
+                            public: false,
+                            collab: false,
+                            desc: desc.to_owned(),
+                        })?;
+                        ui.popup = None;
+                        return Ok(true);
+                    }
+                    _ => {}
+                }
+            }
+        }
+    }
+
+    let command = state
+        .keymap_config
+        .find_command_from_key_sequence(key_sequence);
+    if let Some(Command::ClosePopup) = command {
+        state.ui.lock().popup = None;
+        return Ok(true);
+    }
+
+    let page_type = state.ui.lock().current_page().page_type();
+    match page_type {
+        PageType::Library => page::handle_key_sequence_for_library_page(key_sequence, state),
+        PageType::Search => {
+            page::handle_key_sequence_for_search_page(key_sequence, client_pub, state)
+        }
+        PageType::Context => {
+            page::handle_key_sequence_for_context_page(key_sequence, client_pub, state)
+        }
+        PageType::Browse => {
+            page::handle_key_sequence_for_browse_page(key_sequence, client_pub, state)
+        }
+        #[cfg(feature = "lyric-finder")]
+        PageType::Lyric => {
+            page::handle_key_sequence_for_lyric_page(key_sequence, client_pub, state)
+        }
+    }
```

Comment:
no need to do this for the create playlist popup. Search popup is special because the focus is not placed on it when the popup is created. It's expected to use along with the focused window for interactive searching.

Reply:
just return `Ok(false)` in this case.